options/rtdl/generic/linker.cpp

```diff
@@ -16,6 +16,8 @@ bool stillSlightlyVerbose = false;
 bool logBaseAddresses = false;
 bool eagerBinding = true;
 
+extern DebugInterface debug;
```

Comment:
Can we give that variable a more specific name? Something like `globalDebugInterface`?